hypemaths/models/matrix.py

```diff
@@ -58,8 +58,19 @@ def dims(self) -> tuple:
         """
         return tuple(self._get_mat_dimension(self.matrix))
 
+    @property
+    def size(self) -> int:
+        '''
+        Returns
+        -------
+        int
+            The integer which is the total number of items in the matrix
+        '''
```

Comment:
```suggestion
    def size(self) -> int:
        """
        Returns
        -------
        int
            The integer which is the total number of items in the matrix
        """
```
Use `"` double quotes for eveness